css/types/color.json

```diff
@@ -136,7 +136,7 @@
                 "chrome_android": "mirror",
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": false,
+                  "version_added": "preview",
```

Comment:
Is this in stable in version 128? https://bugzilla.mozilla.org/show_bug.cgi?id=1900251

There are also other relative_syntax features in this file, is more shipping or just relative syntax in `color()`?